spec/lib/efile/md/md502_r_calculator_spec.rb

```diff
@@ -143,6 +149,7 @@
     context 'when filing MFJ with positive federal social security benefits' do
       let(:filing_status) { "married_filing_jointly" }
       before do
+        allow(intake.direct_file_data).to receive(:fed_ssb).and_return(100)
```

Comment:
i had to change this b/c now `:with_spouse` comes after which changes the `fed_ssb` value. But I thought being explicit here that we have a positive fed_ssb anyway is what we want